README.md

```diff
@@ -0,0 +1,53 @@
+# sig-rpc.github.io
+
+The Reasonable Performance Computing SIG's Website & Knowledge-base, where user-submitted guides can help those writing software to identify and address performance problems in their code.
+
+The goal of the website is to make profiling and optimisation guidance accessible to even novice programmers. The main pre-requisite for readers is that they're able to read through code and recognise how it will execute.
+
+### Contributing
+
+Regardless of your familiarity with code performance and optimisation, we welcome your contribution. Contributions may be as small as providing feedback on how a guide is unclear or could be improved, or as large as a fully drafted new guide ready for publication.
+
+Authors with highly technical backgrounds can easily make assumptions, which aren't clear to those less experienced. This is why even feedback from those new to software performance is important!
+
+Ideas and feedback can be submitted via the repository's [issue templates](https://github.com/sig-rpc/sig-rpc.github.io/issues/new/choose).
+
+New articles can be submitted directly as pull requests. It's worth using an existing guide as a template, these can be found in `_profilers/` and `_optimisations/` respectively.
+
+For the full contribution guidance, including a loose style guide, please refer to [`.github/CONTRIBUTING.MD`](.github/CONTRIBUTING.MD).
+
+*Something is better than nothing.*
+
+### Building Locally
+
+The website is a static website built with the Ruby package [Jekyll](jekyllrb).
+
+It's easiest to run Jekyll under a Unix environment, Windows users are encouraged to use the Windows Subsystem for Linux (WSL).
+
+If you don't already have Ruby installed, please refer to Ruby's [installation guide](https://www.ruby-lang.org/en/documentation/installation/).
+
+With Ruby setup, the below command executed in the root of the repository will install the dependencies including Jekyll.
+
+```sh
+bundle install
+```
```

Comment:
```suggestion
```sh
bundle config set path vendor/bundle
bundle install
```
```

> The location on disk where all gems in your bundle will be located regardless of $GEM_HOME or $GEM_PATH values. Bundle gems not found in this location will be installed by bundle install. Defaults to .bundle relative to repository root in Bundler 4, and to the default system path (Gem.dir) before Bundler 4.


I.e. for bundler < 4 this should prevent package conflicts between multiple jekyll websties on a given machine (assuming not also using rbenv or similar?)